### PR TITLE
ADEN | remove unused methods

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsDetection.java [hunk 1]
```diff
@@ -2,12 +2,13 @@
 
 import com.wikia.webdriver.common.core.annotations.NetworkTrafficDump;
 import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
 import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsDetection.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.templates.TemplateNoFirstLoad'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues:
* MINOR 4 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.drivers.Browser'.](https://github.com/Wikia/selenium-tests/blob/8ef8d9c29b5a05cd745e6dc98ac00a169d14d94b/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsOoyalaMercury.java#L4) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.helpers.Emulator'.](https://github.com/Wikia/selenium-tests/blob/8ef8d9c29b5a05cd745e6dc98ac00a169d14d94b/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsOoyalaMercury.java#L5) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)
* MINOR [Remove this unused import 'com.wikia.webdriver.common.core.annotations.InBrowser'.](https://github.com/Wikia/selenium-tests/blob/8ef8d9c29b5a05cd745e6dc98ac00a169d14d94b/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsOoyalaMercury.java#L3) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/common/core/CommonExpectedConditions.java [hunk 2]
```diff
@@ -40,11 +39,7 @@ private CommonExpectedConditions() {
       public Boolean apply(WebDriver from) {
         try {
           String elementsAttributeValue = findElement(locator, from).getAttribute(attribute);
-          if (elementsAttributeValue == null) {
-            return false;
-          } else {
-            return elementsAttributeValue.contains(value);
-          }
+          return elementsAttributeValue != null && elementsAttributeValue.contains(value);
```
Comment (dianafa) [inline src/test/java/com/wikia/webdriver/common/core/CommonExpectedConditions.java]: 👍

**Comment by dianafa** [comment]
so much red <3 LGTM

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsDetection.java (hunk 1 above)
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsDetection.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.templates.TemplateNoFirstLoad'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Pull request merged**